File: java/Mouse.java
public class Mouse extends Peripherals {
    private String mouseName;
    private String status;
    private boolean isWireless;

    public Mouse(String last_device, String last_output, String deviceType, String deviceName, String location, String brand, String mouseName, String status, boolean isWireless) {
        super(last_device, last_output, deviceType, deviceName, location, brand);
        this.mouseName = mouseName;
        this.status = status;
        this.isWireless = isWireless;
    }

    public String getMouseName() {
        return mouseName;
    }

    public String getStatus() {
        return status;
    }

    public String getBrand() {
        return super.getBrand();
    }

    public boolean getIsWireless() {
        return isWireless;
    }

    public void setMouseName(String mouseName) {
        this.mouseName = mouseName;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public void setIsWireless(boolean isWireless) {
        this.isWireless = isWireless;
    }

    public void setLastDevice(String last_device) {
        super.setLastDevice(last_device);
    }

    public void setLastOutput(String last_output) {
        super.setLastOutput(last_output);
    }

    public void setBrand(String brand) {
        super.setBrand(brand);
    }

    public void selfStatus(String status) {
        this.status = status;
    }

    public String info() {
        return "Mouse: " + mouseName + " (Device Type: " + getDeviceType() + ", Device Name: " + getDeviceName() + ", Location: " + getLocation() + ", Brand: " + getBrand() + ", Status: " + getStatus() + ", Wireless: " + (isWireless ? "Yes" : "No") + ", Last Device: " + getLastDevice() + ", Last Output: " + getLastOutput() + ")";
    }
}

File: Python/Mouse.py
from Peripherals import Peripherals


class Mouse(Peripherals):
    def __init__(self, last_device="", last_output="", deviceType="", deviceName="", location="", brand="", mouseName="", status="", isWireless=False):
        super().__init__(last_device, last_output, deviceType, deviceName, location, brand)
        self.mouseName = mouseName
        self.status = status
        self.isWireless = isWireless

    def getMouseName(self):
        return self.mouseName

    def getStatus(self):
        return self.status
    
    def getBrand(self):
        return self.brand

    def getIsWireless(self):
        return self.isWireless

    def setMouseName(self, mouseName):
        self.mouseName = mouseName

    def setStatus(self, status):
        self.status = status

    def setIsWireless(self, isWireless):
        self.isWireless = isWireless

    def setLastDevice(self, last_device):
        self.last_device = last_device

    def setLastOutput(self, last_output):
        self.last_output = last_output

    def setBrand (self, brand):
        self.brand = brand

    def selfStatus(self, status):
        self.status = status

    def info(self):
        return f"Mouse: {self.mouseName} (Device Type: {self.get_device_type()}, Device Name: {self.get_device_name()}, Location: {self.location}, Brand: {self.getBrand()}, Status: {self.getStatus()}, Wireless: {'Yes' if self.isWireless else 'No'}, Last Device: {self.get_last_device()}, Last Output: {self.get_last_output()}"